library/examples/hate_speech.py

```diff
@@ -30,6 +30,7 @@
     test_size=100,
     input_col="text",
     label_col="label",
+    rebalance=False,
```

Comment:
can we rename this? people usually say they under or oversample ( and those are two different mechanisms, I think we undersample based on the code)